shimmingtoolbox/cli/sct_features.py

```diff
@@ -0,0 +1,184 @@
+              help="Method used for extracting the centerline: "
+                   "svm: Automatic detection using Support Vector Machine algorithm."
+                   "cnn: Automatic detection using Convolutional Neural Network."
+                   "viewer: Semi-automatic detection using manual selection of a few points with an interactive viewer "
+                   "followed by regularization. "
+                   "file: Use an existing centerline (use with flag -file_centerline) (default: svm)")
+@click.option('-file_centerline', type=click.Path(), help="Input centerline file (to use with flag -centerline file). "
+                                                          "Example: t2_centerline_manual.nii.gz")
+@click.option('-thr', type=float, required=True,
+              help="Dilate binary or greyscale image with specified size. If shape={'square', 'cube'}: size corresponds"
+                   "to the length of an edge (size=1 has no effect). If shape={'disk', 'ball'}: size corresponds to the"
+                   " radius, not including the center element (size=0 has no effect).")
+@click.option('-brain', type=click.IntRange(0, 1), help="Indicate if the input image contains brain sections (to speed "
+                                                        "up segmentation). Only use with '-centerline cnn'.")
+@click.option('-kernel', type=click.Choice(['2d', '3d']), default='2d',
+              help="Choice of kernel shape for the CNN. Segmentation with 3D kernels is slower than with 2D kernels. "
+                   "(default: 2d)")
+@click.option('-ofolder', type=click.Path(), default='/home/docs/checkouts/readthedocs.org/user_builds/'
+                                                     'spinalcordtoolbox/checkouts/latest/documentation/source)',
+              help="Output folder. Example: My_Output_Folder/ (default: /home/docs/checkouts/readthedocs.org/"
+                   "user_builds/spinalcordtoolbox/checkouts/latest/documentation/source)")
+@click.option('-r', type=click.IntRange(0, 1), default=1, help="Remove temporary files. (default: 1)")
+@click.option('-v', type=click.IntRange(0, 2), default=1, help="1: display on (default), 0: display off, 2: extended "
+                                                               "(default: 1)")
+@click.option('-qc', help="The path where the quality control generated content will be saved")
+@click.option('-qc_dataset', help="If provided, this string will be mentioned in the QC report as the dataset the "
+                                  "process was run on")
+@click.option('-qc_subject', help="If provided, this string will be mentioned in the QC report as the subject the "
+                                  "process was run on")
+@click.option('-igt', help="File name of ground-truth segmentation.")
+def deepseg_sc(fname_input, c, centerline, file_centerline, thr, brain, kernel, ofolder, r, v, qc, qc_dataset, qc_subject, igt):
```

Comment:
deepseg_sc is a cord segmentation algo-- not a morphomath algo

what API will be calling deepseg_sc?